cmd/generate/generate_test.go

```diff
@@ -50,6 +51,22 @@ var (
 		},
 	}
 
+	mcpMaster = &machineconfigurationv1.MachineConfigPool{
```

Comment:
general comment here.

why we are not using ginkgo and gomega like all the other projects?